.github/workflows/test.sh

```diff
@@ -36,8 +38,10 @@ if [ "${maker}" = "cmake" ]; then
     cmake "-DCMAKE_PREFIX_PATH=${lib};${lib64}" ..
 fi
 
+TESTS="s d c z"
+[ "${device}" = "gpu_intel" ] && TESTS="s c"
```

Comment:
I would prefer using an `if-then` block. Conditionally setting a variable in this way seems like a shell hack, rather unintuitive.

Reply:
ok done